wordpress-mobile/AztecEditor-iOS
# Removing NSCodingProxy

### Details:
In this PR we're:

- Updating **CSSAttribute** and **Attribute** so that they're pure swift entities
- Deprecating **NSCodingProxy**
- Conforming to Swift's new **Codable** protocol in **Attribute.Value** and **CSSAttribute**

Needs Review: @diegoreymendez 
Fixes #781

### To test:
1. Verify that the unit test don't break and that there's no more Sanitizer warning
2. Launch the Aztec Demo Document (with sample html!)
3. Select all the text and copy it
4. Kill + Relaunch Aztec (Stop > Run in the sim)
5. Open the Empty Document
6. Paste
7. Verify that Aztec won't crash anymore.
